.github/workflows/build.yml

```diff
@@ -120,7 +114,6 @@ jobs:
           - '3.10'
```

Comment:
I think we should also take the opportunity to use `3.13` instead of `3.13-dev`, right?

Reply:
Good idea